app/models/page.rb

```diff
@@ -177,6 +177,13 @@ def template
     front_matter.fetch(:template, "show")
   end
 
+  # Returns focus keywords to guide content writers with an overview of the page content
+  # Note: it's not for meta keywords, which is a deprecated SEO practice
+  def keywords
+    # Gracefully falls back to the page's path if no keywords are specified to help reduce technical writer workload
+    front_matter.fetch(:keywords) || keywords_from_path
```

Comment:
The second arg of `fetch` is the fallback value so you can write this is the following:

```suggestion
    front_matter.fetch(:keywords, keywords_from_path)
```

Reply:
ah TIL, I've committed the change 👍 

Will merge after checks pass.